app/src/test/java/backgammon/TestPlayerDoublingCube.java

```diff
@@ -0,0 +1,11 @@
+package backgammon;
+
+public class TestPlayerDoublingCube {
+  private boolean cubeOwnership;
+
+  public TestPlayerDoublingCube(boolean x){
+    cubeOwnership = x;
+  }
+
+  public boolean getCubeOwnership(){return this.cubeOwnership;}
+}
```

Comment:
I should have specified this, since I thought it was to be assumed but you should name your tests following the convention [ClassName]Tests.java.

You didn't import the necessary test methods from JUnit, and the `cubeOwnership` member variable for this test class makes no sense.